client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import React from "react";
+import { Link } from "react-router-dom";
 
 const MovieCard = props => {
-  return;
+  const { title, director, metascore, stars } = props.movie;
+  return (
+    <div className="save-wrapper">
+      <Link to={`/movies/${props.movie.id}`}>
+        <div className="movie-card">
+          <h2>{title}</h2>
+          <div className="movie-director">
+            Director: <em>{director}</em>
+          </div>
+          <div className="movie-metascore">
+            Metascore: <strong>{metascore}</strong>
+          </div>
+          <h3>Actors</h3>
+
+          {stars.map(star => (
+            <div key={star} className="movie-star">
+              {star}
+            </div>
+          ))}
+        </div>
+      </Link>
+      <div onClick={props.saveMovie} className="save-button">
```

Comment:
use `button` elements instead of `div` elements to remain semantic and a11y compliant.